src/inet/tests/TestInetLayer.cpp

```diff
@@ -281,14 +281,11 @@ int main(int argc, char * argv[])
 
     while (Common::IsTesting(sTestState.mStatus))
     {
-        struct timeval sleepTime;
         bool lSucceeded = true;
         bool lFailed    = false;
 
-        sleepTime.tv_sec  = 0;
-        sleepTime.tv_usec = 10000;
-
-        ServiceNetwork(sleepTime);
+        constexpr uint32_t kSleepTimeMilliseconds = 100;
```

Comment:
```suggestion
        constexpr uint32_t kSleepTimeMilliseconds = 10;
```
Again, to match old behavior.